### PR TITLE
Remove unneccessary defer.

### PR DESCRIPTION
**Pull request by mbrandonw**
We got a good [tip](https://twitter.com/rmayoff/status/1387560179340914694) from twitter that the `defer` in `onChange` doesn't seem to be necessary.

### CODE REVIEW

File: Sources/AppFeature/AppView.swift [hunk 1]
```diff
@@ -450,8 +450,8 @@ public struct AppView: View {
               appState.game.map {
                 (
                   game: $0,
-                  nub: nil,
-                  settings: .init(
+                  nub: CubeSceneView.ViewState.NubState?.none,
+                  settings: CubeSceneView.ViewState.Settings(
```
Comment (mbrandonw) [inline Sources/AppFeature/AppView.swift:454]: gonna go ahead and fix this

**Comment by mbrandonw** [comment]
I also went ahead and updated App Store screenshots since #101 merged.

**Comment by stephencelis** [comment]
Thanks! Yeah I guess we may have used that long ago?

**Comment by mbrandonw** [comment]
Yeah think so, but having a hard time remembering the early versions of the operator.

**Comment by mluisbrown** [comment]
Oh nice 👍  I was just about to ask if there was something I was missing about the `defer` 😄

**Pull request merged**